server/views/pages/curfewAddress/checkYourAnswers.njk

```diff
@@ -0,0 +1,74 @@
+{% extends "layout.njk" %}
+{% from "partials/offenderSummary.njk" import offenderSummary %}
+{% from "partials/assessment/addressSummary.njk" import addressSummary %}
+{% from "partials/assessment/residentSummary.njk" import residentSummary %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - Check your answers" %}
+{% set backLinkHref = "/" %}
+{% set addressCount = addressSummary.length %}
+{% set preferedAddress = checkRequestsForassessmentSummary | filter('preferencePriority', 'FIRST') %}
+{% set secondAddress = checkRequestsForassessmentSummary | filter('preferencePriority', 'SECOND') %}
```

Comment:
**suggestion:** you can `find` these so you don't have to deal with them as arrays